include/sot/core/device.hh

```diff
@@ -93,6 +93,7 @@ namespace dynamicgraph {
       void setVelocitySize(const unsigned int& size);
       virtual void setVelocity(const dg::Vector & vel);
       virtual void setSecondOrderIntegration();
+      virtual void setNoIntegration();
```

Comment:
The method setNoIntegration seems redundant with respect to the method setControlInputType, which can be used to specify the control input type CONTROL_INPUT_NO_INTEGRATION